README.md

```diff
@@ -31,11 +31,34 @@ Family context expects a process similar to the following to be in place:
 ```
 
 ## Setup
-The easiest way to run this is to do the following commands after you have 
+### Running Locally for Development
+The following steps can be used to run the service in a way that is easy to develop, keeping the database 
+locally and the service easily able to update when changes are made.
+1. Install the libraries needed for the project
+   1. `poetry install`
+   2. Copy the file `.env.sample` to `.env` and fill out the values. If you want to use SQLite, then 
+   delete the database variables
+   3. `poetry run python manage.py migrate` to setup the database
+2. Build the front end 
+   1. `cd frontend`
+   2. `npm install`
+   3. `npm run build`
```

Comment:
side note: Is it intentional that this project uses npm instead of yarn?
yarn has better caching so faster runtimes, I think.

Reply:
I had no preference either way. They were just the instructions I was given from the django template.